src/app/routes/applications/applications.component.ts

```diff
@@ -65,6 +65,13 @@ export class ApplicationsComponent implements OnInit, AfterViewInit, OnDestroy {
   }
 
   openNewOrgComponent(): void {
+    if (!this.isIamEwcOwner) {
+      const namespace = 'orgowner.roles.orgcreator.apps.energyweb.iam.ewc';
```

Comment:
I think `namespace` should just be the app name? Not sure about that though, you would need to test to confirm
```suggestion
      const namespace = 'orgcreator.apps.energyweb.iam.ewc';
```